metadata-db-app/int_test/cmr/metadata_db/int_test/utility.clj

```diff
@@ -1,22 +1,18 @@
 (ns cmr.metadata-db.int-test.utility
   "Contains various utility methods to support integration tests."
-  (:require [clojure.test :refer :all]
-            [clojure.string :as str]
-            [clj-http.client :as client]
-            [cheshire.core :as json]
-            [clojure.edn :as edn]
-            [clojure.walk :as walk]
-            [clj-time.core :as t]
-            [clj-time.format :as f]
-            [clj-time.local :as l]
-            [clj-time.coerce :as cr]
-            [inflections.core :as inf]
-            [cmr.common-app.test.side-api :as side]
-            [cmr.common.util :as util]
-            [cmr.metadata-db.config :as config]
-            [clj-http.conn-mgr :as conn-mgr]
-            [cmr.transmit.config :as transmit-config]
-            [cmr.metadata-db.config :as mdb-config]))
+  (:require
+   [cheshire.core :as json]
+   [clj-http.client :as client]
+   [clj-http.conn-mgr :as conn-mgr]
+   [clj-time.format :as f]
+   [clojure.string :as string]
+   [clojure.test :refer :all]
+   [cmr.common-app.test.side-api :as side]
+   [cmr.common.util :as util]
+   [cmr.metadata-db.config :as mdb-config]
+   [cmr.metadata-db.services.concept-service :as concept-service]
+   [cmr.transmit.config :as transmit-config]
+   [inflections.core :as inf]))
```

Comment:
Ah, I love seeing ns cleanup like this :-) Removing unused `require`s is good for one's (code) health ...